### PR TITLE
Add Nocode summit France 2024

### PR DESCRIPTION
**Pull request by lapasque**
Add the nocode summit France in October

### CODE REVIEW

**Comment by lapasque** [comment]
Add nocode summit France 2024

File: README.md [hunk 1]
```diff
@@ -476,7 +476,8 @@ All the data (past and coming) are available publicly in JSON:
...
 * 16-17: [BaselOne 2024](https://baselone.ch) - Basel (Switzerland) <a href="https://sessionize.com/baselone-2024"><img alt="CFP BaselOne 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2001-April-2024&color=red"></a>
-* 16-17: [#HelloStavanger 2024](https://hellostavanger.no) - Stavanger (Norway) <a href="https://sessionize.com/hellostavanger-2024"><img alt="CFP #HelloStavanger 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2007-April-2024&color=red"></a>
+* 16-17: [#HelloStavanger 2024](https://hellostavanger.no) - Stavanger (Norway) <a href="https://www.nocode-summit.org/fr/speakers"><img alt="CFP NoCodeSummit 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2007-April-2024&color=red"></a>
+* 16-17: [NoCode Summit 2024](https://www.nocode-summit.org/fr) - Paris (France) <a href="https://sessionize.com/hellostavanger-2024"><img alt="CFP #HelloStavanger 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%20&color=green"></a>
```
Comment (scraly) [inline README.md:480]:
```suggestion
* 16-17: [NoCode Summit 2024](https://www.nocode-summit.org/fr) - Paris (France)
```

**Review by scraly — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -476,7 +476,8 @@ All the data (past and coming) are available publicly in JSON:
...
 * 16-17: [Camunda Con New York](https://www.camundacon.com/na/) - New York, NY (USA) & Online <a href="https://www.camundacon.com/na/call-for-presentations/"><img alt="CFP Camunda Con New York" src="https://img.shields.io/static/v1?label=CFP&message=until%2029-May-2024&color=red"></a>
 * 16-17: [BaselOne 2024](https://baselone.ch) - Basel (Switzerland) <a href="https://sessionize.com/baselone-2024"><img alt="CFP BaselOne 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2001-April-2024&color=red"></a>
-* 16-17: [#HelloStavanger 2024](https://hellostavanger.no) - Stavanger (Norway) <a href="https://sessionize.com/hellostavanger-2024"><img alt="CFP #HelloStavanger 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2007-April-2024&color=red"></a>
+* 16-17: [#HelloStavanger 2024](https://hellostavanger.no) - Stavanger (Norway) <a href="https://www.nocode-summit.org/fr/speakers"><img alt="CFP NoCodeSummit 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2007-April-2024&color=red"></a>
```
Comment (scraly) [inline README.md:479]:
```suggestion
* 16-17: [#HelloStavanger 2024](https://hellostavanger.no) - Stavanger (Norway) <a href="https://sessionize.com/hellostavanger-2024"><img alt="CFP #HelloStavanger 2024" src="https://img.shields.io/static/v1?label=CFP&message=until%2007-April-2024&color=red"></a>
```

**Review by scraly — CHANGES_REQUESTED** [review]

**Comment by scraly** [comment]
Thx
I fixed two times your suggestions :)

**Pull request merged**